sandbox/settings.py

```diff
@@ -213,6 +213,12 @@ class Base(MagnifyCoreConfigurationMixin, Configuration):
         environ_prefix=None,
     )
 
+    USERNAME_REGEX = values.Value(
+        r"^[a-z0-9_.-]+$",
+        environ_name="MAGNIFY_USERNAME_REGEX",
+        environ_prefix=None,
```

Comment:
I wonder if, by default, Keycloak is not less stricter about username allowed characters. Shouldn't we use the default username keycloak validator as default value otherwise a user could register on Keycloak then raise a ValidationError when requesting api for the first time ?

Reply:
yes you're right... we need to figure this out then :thinking:

Reply:
I fixed keycloak's config in commit https://github.com/openfun/jitsi-magnify/pull/198/commits/bf2251024491cdd4cfa57aa2936926eaab30de17

Reply:
It works well! I just don't know how to apply realm changes without resetting my db, but I think cli is able to do that.